setup.py

```diff
@@ -19,6 +19,7 @@
     author="Pavel Dedík",
     author_email="pavel.dedik@kiwi.com",
     packages=find_packages(),
+    python_requires=">='3.5'",
```

Comment:
Shouldn't this be -
```suggestion
    python_requires=">=3.5",
```

I'm not sure it will work otherwise, just to be sure.

Reply:
Fixed!

Reply:
That was fast, thx!